utilities/rocAL/rocAL_dataloader_mt/rocAL_dataloader_mt.cpp

```diff
@@ -157,47 +149,48 @@ int thread_func(const char *path, int gpu_mode, RocalImageColor color_format, in
             break;
         // copy output to host as image
         rocalCopyToOutput(handle, mat_input.data, h*w*p);
-        rocalGetImageLabels(handle, labels.data());
-        int img_name_size = rocalGetImageNameLen(handle, image_name_length);
+        RocalTensorList labels = rocalGetImageLabels(handle);
+        unsigned img_name_size = rocalGetImageNameLen(handle, image_name_length);
         char img_name[img_name_size];
         rocalGetImageName(handle, img_name);
 #if PRINT_NAMES_AND_LABELS
         std::string imageNamesStr(img_name);
         int pos = 0;
-        for(int i = 0; i < inputBatchSize; i++)
+        int *labels_buffer = reinterpret_cast<int *>(labels->at(0)->buffer());
+        for(int i = 0; i < batch_size; i++)
         {
-            names[i] = imageNamesStr.substr(pos, ImageNameLen[i]);
-            pos += ImageNameLen[i];
-            std::cout << "name: " << names[i] << " label: "<< labels[i] << " - ";
+            names[i] = imageNamesStr.substr(pos, image_name_length[i]);
+            pos += image_name_length[i];
+            std::cout << "name: " << names[i] << " label: "<< labels_buffer[i] << " - ";
         }
         std::cout << std::endl;
 #endif
         iter_cnt ++;
 
         if(!display)
             continue;
-        mat_input.copyTo(mat_output(cv::Rect(  col_counter*w, 0, w, h)));
+        mat_input.copyTo(mat_output(cv::Rect(col_counter * w, 0, w, h)));
         cv::cvtColor(mat_output, mat_color, CV_RGB2BGR);
         if(DISPLAY)
         cv::imshow("output.png",mat_output);
         else
-        cv::imwrite("output.jpg",mat_output);
+        cv::imwrite("output.png",mat_output);
 
         col_counter = (col_counter+1)%number_of_cols;
     }
 
     high_resolution_clock::time_point t2 = high_resolution_clock::now();
     auto dur = duration_cast<microseconds>( t2 - t1 ).count();
     auto rocal_timing = rocalGetTimingInfo(handle);
-    std::cout << "Load     time:"<< shard_id<<" " <<rocal_timing.load_time << std::endl;
-    std::cout << "Decode   time:"<< shard_id<<" " <<rocal_timing.decode_time << std::endl;
-    std::cout << "Process  time: "<< shard_id<<" " <<rocal_timing.process_time << std::endl;
-    std::cout << "Transfer time: "<< shard_id<<" " <<rocal_timing.transfer_time << std::endl;
+    std::cout << "For shard_id: " << shard_id << std::endl;
```

Comment:
@swetha097 Are we taking shards as threads. Please say thread id then, since it's a multithreaded exmaple. Users will be confused

Reply:
JPEG data loader are based on numshards and shard_id.
That is the shard_id outputs added in begenning instead of adding it after the Load Time, Decode Time, Process and Transfer times